src/raftstore/store/peer_storage.rs

```diff
@@ -75,6 +75,35 @@ impl PartialEq for SnapState {
     }
 }
 
+// Discard all log entries prior to compact_index. We must guarantee
+// that the compact_index is not greater than applied index.
+pub fn compact_raft_log(tag: &str,
```

Comment:
It do compact raft log by updating meta data, so old raft log can't be accessed any more. It just doesn't actually delete old logs.